exo-go/features/fetch_templates/fetch_templates.feature

```diff
@@ -0,0 +1,13 @@
+Feature: fetching updates for existing templates
+
+  When developing an exosphere application
+	I want to be able to fetch the latest updates for the existing service templates in application codebase
+	So that I don't have to manually run the git command to update the submodules
+
+  - run "exo fetch-templates" to add a remote service template to the application codebase
+
+  Scenario: fetching updates for all existing service templates
+    Given I am in the root directory of an empty git application repository called "test app"
+    When running "exo add-template boilr-spark https://github.com/tmrts/boilr-spark.git" in my application directory
+    And running "exo fetch-templates" in my application directory
+    Then it prints "done" in the terminal
```

Comment:
The `running` step calls `cmd.CombinedOutput()` which runs the command, waits for the process to finish and returns its combined output. So if that step didn't fail, the process didn't exit with non-zero code. Getting the exit code, based on [this](https://github.com/Originate/exosphere/blob/cm-remote-boilr-templates/exo-go/test_helpers/shared_feature_context.go#L152), requires checking the `err` returned by `cmd.Wait()`.  Since we don't call `cmd.Wait()`, we will need to a lot more work to get the exit code. I could try to do that if you want.